doc/motion_planning_api/src/motion_planning_api_tutorial.cpp

```diff
@@ -123,8 +138,11 @@ int main(int argc, char** argv)
   // The package MoveItVisualTools provides many capabilties for visualizing objects, robots,
   // and trajectories in RViz as well as debugging tools such as step-by-step introspection of a script
   namespace rvt = rviz_visual_tools;
-  moveit_visual_tools::MoveItVisualTools visual_tools("panda_link0");
-  visual_tools.deleteAllMarkers();
+  moveit_visual_tools::MoveItVisualTools visual_tools("panda_link0", rviz_visual_tools::RVIZ_MARKER_TOPIC, psm);
```

Comment:
I believe this worked quite well without the `PlanningSceneMonitor` before.
Why do you propose to add the whole overhead of setting up an updating scene monitor above?

Reply:
`PlanningSceneMonitor` is used to update the robot state in the planning scene to the `initial` configuration in the [fake_controllers.yaml](https://github.com/ros-planning/panda_moveit_config/blob/8ef20042c82194d2c94da955dfea4d8517ffa6b1/config/fake_controllers.yaml#L15). The old setup worked because the default configuration of the PR2 wasn't in self-collision as is the case with the Panda